second-import.wdl

```diff
@@ -1,6 +1,12 @@
+<<<<<<< HEAD
```

Comment:
Merge conflict is still here for me, please confirm

Reply:
Fixed